MockServer/src/main/java/com/github/dedis/student20_pop/mockserver/SimpleEncoder.java

```diff
@@ -0,0 +1,18 @@
+package com.github.dedis.student20_pop.mockserver;
+
+import javax.websocket.Encoder;
+import javax.websocket.EndpointConfig;
+
+public class SimpleEncoder implements Encoder.Text<String> {
```

Comment:
Same here, a comment would be nice

Reply:
This class was not needed so I removed it